class/defaults.yml

```
@@ -1,3 +1,11 @@
 parameters:
   nfs_client_provisioner:
-    namespace: syn-nfs-client-provisioner
+    namespace: syn-storage
+    server: nfs.local
+    path: /srv/nfs/export
+    mountOptions: null
+    storageClass:
+      defaultClass: true
```

This is dangerous. We have https://github.com/projectsyn/component-storageclass/blob/master/lib/storageclass.libsonnet to ensure there is only one StorageClass with the default annotation.

At least make it default to `false`.

```suggestion
      defaultClass: false
```

---

removed storageclass config completely as requested by @srueg